src/styles/docTemplate.css

```diff
@@ -27,6 +27,10 @@
       list-style: initial;
     }
 
+    li {
+      margin-top: 8px;
+    }
```

Comment:
Why?

Reply:
be close to Figma.

from:
![image](https://github.com/user-attachments/assets/090417dc-ee66-43a3-94f3-da49ef41e0f3)
->
![image](https://github.com/user-attachments/assets/20be9443-7bc9-4531-b3db-a6b37c8d27c5)

Figma:
![image](https://github.com/user-attachments/assets/14dfa422-85d2-4c79-8878-7c13a099aad0)